deploy/examples/create-external-cluster-resources.py

```diff
@@ -1244,10 +1247,14 @@ def get_rgw_fsid(self):
         secret_key = self.out_map["RGW_ADMIN_OPS_USER_SECRET_KEY"]
         rgw_endpoint = self._arg_parser.rgw_endpoint
         cert = None
-        verify = None
-        if self._arg_parser.rgw_tls_cert_path and not self._arg_parser.rgw_skip_tls:
-            cert = self.validate_rgw_endpoint_tls_cert()
-            verify = True
+        verify = True
+        if (
+            self._arg_parser.rgw_tls_cert_path
+            and not self._arg_parser.rgw_skip_tls
+            and self.validate_rgw_endpoint_tls_cert()
+        ):
```

Comment:
```suggestion
        if (
            self.validate_rgw_endpoint_tls_cert()
            and not self._arg_parser.rgw_skip_tls
        ):
```
The check for `self._arg_parser.rgw_tls_cert_path` is redundant, because the same check is in the beginning of the `validate_rgw_endpoint_tls_cert` method.

Reply:
DOne